src/lib/components/copy/Copier.tsx

```diff
@@ -24,15 +25,15 @@ export const Copier = ({
     value={value}
     copyLabel={copyLabel}
     triggerElement={
-      <CopyIcon
+      <Flex
         className={className}
         display={display}
-        boxSize="16px"
-        color="text.dark"
         cursor="pointer"
         marginLeft={ml}
         onClick={() => AmpTrack(AmpEvent.USE_COPIER)}
-      />
+      >
+        <CustomIcon name="copy" boxSize="16px" />
+      </Flex>
```

Comment:
because of the requested change in `CustomIcon`, move all properties from `Flex` to `CustomIcon`